.github/workflows/fmt_build.yml

```diff
@@ -0,0 +1,26 @@
+name: cargo-gccrs CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      master
+
+jobs:
+  fmt-and-build:
```

Comment:
Format-lint-build-test or something might hint that it is also running unit-tests and clippy

Reply:
Good point. I moved all testing to the other file, so that is unit testing and integration testing. This workflow only runs lints and compilation checks